### ISSUE TITLE
Light mode issues

### ISSUE THREAD

**Issue by IzKuipers** [ArcOS-Project/v7#281]
I've noticed several issues with our current light mode implementation:
- the calendar segments have a shadow in light mode
- `<a>`s and `<button class="link">`s aren't very legible in many accent colors in light mode
- The blurred profile picture on the account page of settings is too visible in light mode
-

**Comment by Blockyheadman** [comment]
I've made some.. *light* changes for how light mode is treated. CSS only changes.

- Firstly, the profile picture blur in the accounts screen is now slightly more transparent, now set to `.2` instead of it's default `0.3`
- Second, the calendar shadows now use the `--secondary-box-shadow` CSS variable instead of a hard-coded value
- And lastly, the `button` and `a` elements that have accent color applied now use `--accent-suggested-end` with `--accent-dark` as a fallback.

Regarding the last change, there's only so much I can do with a CSS only change and some burnout so it doesn't guarantee a *perfect* coverage for dummies, but that's too much to consider.
So just, **don't** use white or things suuuper close to white without considering the possibility of legibility.

<img width="464" height="498" alt="Image" src="https://github.com/user-attachments/assets/49b3d9de-aaed-4bea-b0ff-d01e486ad747" />

<img width="233" height="251" alt="Image" src="https://github.com/user-attachments/assets/142610a6-fc4e-4f41-a634-3733fec72352" />

But I will say, this does help tremendously so I believe it should be applied elsewhere, like the logo in this example or even the icons in the file manager.

I'm so tired lol.
